packages/codemods/fastify/4/expose-head-routes/README.md

```diff
@@ -0,0 +1,36 @@
+This codemod makes exposeHeadRoutes true by default
+
+🚦 **Impact Level**: Minimal
+
+## What Changed
+
+Starting with v4, every GET route will create a sibling HEAD route. You can revert this behavior by setting exposeHeadRoutes: false in the server options.
+
+## Before
+
+```jsx
+const fastify = require('fastify')();
+
+fastify.get('/example', (request, reply) => {
+    reply.send({ message: 'Hello, World!' });
+});
+
+fastify.listen(3000, () => {
+    console.log('Server is running on http://localhost:3000');
+});
+
+```
+
+## After
+
+```jsx
+const fastify = require('fastify')();
+
+fastify.get('/example', (request, reply) => {
+    reply.send({ message: 'Hello, World!' });
+});
+fastify.listen(3000, () => {
+    console.log('Server is running on http://localhost:3000');
+});
+
+```
```

Comment:
before and after look identical to me

Reply:
this is manual change and we might not need a codemod for this. this is not required.